portal/src/components/FilterCategories/Search/Search.component.less

```diff
@@ -68,38 +45,6 @@
       list-style: none;
       display: inline-block;
     }
-
-    @media @mobile {
```

Comment:
We hebben nog geen toestemming om themes eruit te slopen

Reply:
De themes werken nog wel, ik heb alleen het mobile deel eruit gesloopt aangezien dat toch niet aan staat. Als we het wel weer gaan supporten moeten we het maar goed doen.